### PR TITLE
formatted styles added

### PR DESCRIPTION
**Pull request by Maly22**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,9 +15,12 @@
...
    <header class="header">
     <nav id="navegation"> 
-      <a class="header__menu__link" href="#About"> About</a></li>
-      <a class="header__menu__link" href="#Eperience">Experience</a></li>
-      <a class="header__menu__link" href="#Connect">Connect</a></li>
+      <a class="header__menu__link" href="#About"> About</a>
+      <a class="header__menu__link" href="#Experience">Experience</a>
+      <a class="header__menu__link" href="#Connect">Connect</a>
+      <a class="header__menu__link" href="#Skills">Skills</a>
+      <a class="header__menu__link" href="#Projects">Projects</a>
```
Comment (brandaogabriel7) [inline index.html:23]: You didn't use capital letters for the skills and projects sections. The id is case sensitive so you need to update your sections to use `Skills` and `Projects` as their ids. Or you can update the links in your `nav` to use the current ids, which are `skills` and `projects`.

Comment (brandaogabriel7) [reply index.html:23]: I suggest keeping the first letters capitalized to follow the same convention as the other sections.

File: css/index.css
Comment (brandaogabriel7) [inline css/index.css:1]: Be sure to keep indentation consistent in your files. You can use the VSCode command "Format the document" to achieve that.

**Review by brandaogabriel7 — APPROVED** [review]
Nice job, Marilyn! Just be sure to fix the issue with your sections ids and keep indentation consitent.

File: css/index.css
Comment (Maly22) [reply css/index.css:1]: Thanks, I did it.

**Pull request merged**